narwhal/executor/src/lib.rs

```diff
@@ -42,12 +41,7 @@ pub type SerializedTransactionDigest = u64;
 // Important - if you add method with the default implementation here make sure to update impl ExecutionState for Arc<T>
 pub trait ExecutionState {
```

Comment:
At this point we can change `ExecutionState::load_execution_indices` to only return u64 with `last_committed_round` right? And ExecutionIndexes at this point can be entirely moved to sui